index.js

```diff
@@ -67,18 +67,20 @@ module.exports = {
             headers['Authorization'] = 'token ' + token;
           }
 
-          var options = {
-            method: 'POST',
-            uri: 'https://api.github.com/repos/' + org + '/' + repo + '/deployments',
-            headers: headers,
-            body: body,
-            json: true
+          this.log('willDeploy | token:', token, { verbose: true});
```

Comment:
@balinterdi This is the signature of `this.log` -> https://github.com/ember-cli-deploy/ember-cli-deploy-plugin/blob/1943a13e237e94e23b75a3fdc9b56dd04799d545/index.js#L79

Try changing this to:

```suggestion
          this.log(`willDeploy | token: ${token}`, { verbose: true});
```

Make the same change on the other calls too. See if that fixes things

Reply:
Thanks for pointing that out 🤦🏼‍♂️ 

Unfortunately, that didn't seem to have mattered, I still don't see anything logged. I also compared the CI job output for a branch that uses the old version of this library with the one that uses this branch and saw no significant diff.

I double checked that `axios.request` returns a promise as advertised [and it does](https://github.com/axios/axios/blob/master/lib/core/Axios.js#L29-L92) so I'm running out of ideas.

Is there a way to trigger the ember-cli-deploy hooks locally? (outside of the deployment process)

Reply:
Yeh, use the `pipeline.disabled.allExcept` config variable to disable all plugins except this one. Then run the deployment locally.

Reply:
@balinterdi http://ember-cli-deploy.com/docs/v1.0.x/configuration/#disabling-plugins

Reply:
Thank you, that was a great tip.

I managed to fix an error that way and everything seems to work locally. However, when a deployment is requested as part of the PR, it seems to be stuck, as if the deployment status hadn't been correctly updated:

<img width="875" alt="Screenshot 2022-02-03 at 11 31 12" src="https://user-images.githubusercontent.com/5022/152325682-bb8f93a9-c07d-442c-b434-a61aaa8410a4.png">

Do you have a hunch about where this can go wrong?